src/main/kotlin/org/rust/lang/core/completion/LookupElements.kt

```diff
@@ -191,8 +191,14 @@ fun LookupElementBuilder.toKeywordElement(keywordKind: KeywordKind = KeywordKind
     toRsLookupElement(RsLookupElementProperties(keywordKind = keywordKind))
 
 private fun RsElement.getLookupElementBuilder(scopeName: String, subst: Substitution): LookupElementBuilder {
+    val isProcMacroDef = this is RsFunction && isProcMacroDef
+    val icon = when {
+        this is RsFile -> RsIcons.MODULE
+        isProcMacroDef -> RsIcons.MACRO
+        else -> getIcon(0)
+    }
     val base = LookupElementBuilder.createWithSmartPointer(scopeName, this)
-        .withIcon(if (this is RsFile) RsIcons.MODULE else this.getIcon(0))
+        .withIcon(icon)
```

Comment:
We could instead tune `RsFunctionImplMixin.getIcon()` in order to show the macro icon in other cases as well (structure view)